packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -128,6 +130,29 @@ export const getInterpolation = <TNode extends {}>(
     const { accessPath, bindingIdentifier } = getMemberExpressionMeta(expression);
     const binding = meta.parentPath.scope.getBinding(bindingIdentifier.name);
 
+    if (
+      binding &&
+      binding.path.isImportDefaultSpecifier() &&
+      binding.path.parentPath.isImportDeclaration()
+    ) {
+      const moduleImportName = binding.path.parentPath.node.source.value;
+      const modulePath = require.resolve(meta.state.cwd + '/' + moduleImportName);
```

Comment:
Where we are setting `meta.state.cwd` 🤔?

Reply:
babel sets it - we override it in the test https://github.com/atlassian-labs/compiled/pull/266/files#diff-7162f4ecb36d5e9e278ec6b3e335af9eR13

still need to figure out what the best path is for all of this, will flesh it out more when i start doing fake node modules

Reply:
Are we covering imports with aliases or paths like we have in webpack or tsconfig? 🤔

for eg:
`import fn from '@root/utils/fn';`

or

`import fn from 'src/utils/fn';`

Reply:
probably. any suggestions? it's really only a problem in jira - in AF its just relative + node modules

Reply:
the pragmatic approach can also be "solve this later" if we need

Reply:
hmm..may be we can pass something as babel option. Like we have this in `eslint-plugin-import`
https://github.com/benmosher/eslint-plugin-import#resolvers